/**:
  processor:
    ros__parameters:
      hamming_distance: 19
      marker_width: 0.115
      dfov: 68.5 #get from camera info?
      hfov: -0.0
      use_rolling_filter: true
      rolling_filter_len: 15
      label_color_image: true
      label_depth_image: false

  calibrator:
    ros__parameters:
      calibration_config_file: 'package://stag_ros2/config/stag_markers.yaml'
      trigger_calibration_once: False
      minimum_calibration_markers: 2

  mqtt_forwarder:
    ros__parameters:
      mqtt_encoding: 'json'
      mqtt_ns: 'stag_ros2'
